spec.bs

```diff
@@ -338,7 +363,9 @@ const request = new PaymentRequest([{
       icon: "https://fancybank.com/card-art.png",
     }, 
 
-    timeout: 360000,  // 6 minutes
+Q: Sugest we define that displayName is Mandatory and icon is optional?
```

Comment:
To be clear, you're suggesting a change to https://w3c.github.io/secure-payment-confirmation/#sctn-paymentcredentialinstrument-dictionary ?

I'm not sure offhand whether icon should be optional. Currently all users of SPC wish to pass an icon, and it's much easier to go required --> optional than the other way around, so I would by default lean to keeping both required.